db/migrate/20220326105838_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :Photo
+      t.text :Bio
+      t.integer :PostsCounter
```

Comment:
- [ ] Kindly note that it's always a good practice to name all columns in a Postgres table in lowercase and separate words with `_`. example `posts_pounter` instead of `PostsCounter`, `name` instead of `Name` .... Kindly rename these table's column accordingly.